### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by ocaballeror**
Sorry for the unintuitive name, but I didn't find a better name for all these things.

This is basically a list of things that I would find useful, but feel free to criticize the ones you don't like. The only one that fixes a bug is 1d97dac , the others are just nice things to have.

Btw, thanks for this tool. I've been using it for a while 😄

### CODE REVIEW

**Comment by intgr** [comment]
Thanks! I'll try to take a look within a few days.

File: topy/topy.py [hunk 1]
```diff
@@ -56,7 +57,12 @@ def load_rules(filename):
 
     with open(filename) as rulefile:
         # Use html.parser: lxml is only slightly faster & requires an additional dependency.
```
Comment (intgr) [inline topy/topy.py:59]: This comment needs updating as well.

File: topy/topy.py [hunk 2]
```diff
@@ -75,7 +81,7 @@ def load_rules(filename):
         try:
             r = regex.compile(find)
             # Use \1 instead of $1 etc
-            replace = replace.replace('$', '\\')
+            replace = regex.sub(r'\$(.)', r'\\1', replace)
```
[the remark this answers is not in the document]
Comment (ocaballeror) [reply topy/topy.py:84]: Very true. That explains why the `-tility` rule was replacing "utility" with "utilitu".

I'll move it to a function as you suggest.

**Comment by ocaballeror** [comment]
@intgr Sorry for the force push. I had to update almost all the commits, and also I wanted to drop the one removing the '-utility' rule, now that I know it's not bugged and the issue is my code instead 😞

File: README.rst [hunk 3]
```diff
@@ -29,6 +29,8 @@ Usage::
       -a, --apply           overwrite files in place
       -r FILE, --rules=FILE
                             specify custom ruleset file to use
+      -x RULE, --exclude=RULE
```
Comment (intgr) [inline README.rst:32]: Update README too. :)

Comment (ocaballeror) [reply README.rst:32]: Oops 😅

**Comment by intgr** [comment]
Thanks @ocaballeror, will make a release soon.

By the way, I want to drop Python 2 support after this release, I hope you won't be missing it?

**Comment by ocaballeror** [comment]
Definitely not. Everything related to removing python 2 is a win in my books. 

Thanks for merging!

**Comment by intgr** [comment]
Released version 0.3.0 with your changes on PyPI and GitHub: https://github.com/intgr/topy/releases/tag/v0.3.0

**Pull request merged**